Formula/qt.rb

```diff
@@ -69,6 +69,10 @@ def install
       end
     end
 
+    if MacOS.version >= :sierra
+      args << "-no-phonon"
+    end
```

Comment:
You can inline this:
```rb
args << "-no-phonon" if MacOS.version >= :sierra
```

You can also add a short comment above the line to explain why it’s here.